src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/get_system_info_response_test.cc

```diff
@@ -131,6 +131,20 @@ TEST_F(GetSystemInfoResponseTest, GetSystemInfo_UNSUCCESS) {
   command->Run();
 }
 
+TEST_F(GetSystemInfoResponseTest, GetSystemInfo_Comparison_ccpuVersion) {
+  MessageSharedPtr command_msg = CreateCommandMsg();
+  (*command_msg)[strings::params][hmi_response::code] =
+      hmi_apis::Common_Result::SUCCESS;
+  (*command_msg)[strings::msg_params][hmi_response::capabilities] =
+      (capabilities_);
+
+  ResponseFromHMIPtr command(CreateCommand<GetSystemInfoResponse>(command_msg));
+
+  EXPECT_CALL(mock_hmi_capabilities_, matches_ccpu_version(ccpu_version));
+
+  command->Run();
```

Comment:
@ydementieiev Before run a command you need to initialize it as well. So, please add `ASSET_TRUE(command_->Init())`

Reply:
@IGapchuk  done [faf31b](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/faf31b567f45e2c8d77bec815805113a0b34ba63)